test/nginx-docker-compose.yaml

```
@@ -6,7 +6,10 @@ services:
     image: macbre/nginx-brotli:1.19.3
```

```suggestion
    image: macbre/nginx-brotli:1.19.4
```

This can solve SSL issues (1.19.3 image had an issue with nginx build) - see https://github.com/macbre/docker-nginx-brotli/pull/15

---

@gmetais - no we have

```
 localhost but https version
    ✗ should be generated 
        » No error should be thrown: got Error: net::ERR_CERT_AUTHORITY_INVALID at https://127.0.0.1:8889 // /home/runner/work/phantomas/phantomas/node_modules/vows/lib/assert/macros.js:31
```

We need to pass a different option to puppeteer.

---

FIxed in #821